crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -893,6 +893,28 @@ impl TypedTransaction {
     }
 }
 
+impl From<alloy_rpc_types::Transaction> for TypedTransaction {
+    fn from(transaction: alloy_rpc_types::Transaction) -> Self {
+        // TODO(serge): Implement or delete based on feedback. Below is a hack
+        let legacy = TxLegacy {
```

Comment:
@mattsse thoughts on this conversion? Is this my only choice? Mapping out all the transaction types and converting them here based on the RPC transaction?